src/index.js

```diff
@@ -101,6 +101,9 @@ export default class ReactPdfJs extends Component {
     }
     const viewport = page.getViewport(scale);
     const { canvas } = this;
+    if (!canvas) {
```

Comment:
can we move this to line 98 and say 

```js
if (!this.cavas) return;
```

Will make more sense up there since this is a validation check